ddtrace/contrib/aiopg/__init__.py

```diff
@@ -0,0 +1,28 @@
+"""Instrument aiopg to report Postgres queries.
+
+``patch`` will automatically patch your aiopg connection to make it work.
+::
+
+    from ddtrace import Pin, patch
+    import aiopg
+
+    # If not patched yet, you can patch aiopg specifically
+    patch(aiopg=True)
+
+    # This will report a span with the default settings
+    async with aiopg.connect(DSN) as db:
+        with (await db.cursor()) as cursor:
+            await cursor.execute("select * from users where id = 1")
+
+    # Use a pin to specify metadata related to this connection
+    Pin.override(db, service='postgres-users')
+"""
+from ..util import require_modules
+
+required_modules = ['aiopg']
+
+with require_modules(required_modules) as missing_modules:
+    if not missing_modules:
+        from .patch import patch
+
+        __all__ = ['patch']
```

Comment:
Don't know if we want to export more stuff here, I think that the `patch()` method for those integrations is the way to go, rather than exporting extra functions that people can add manually in their code. What do you think?